boards/arm/actinius_icarus_bee/Kconfig.defconfig

```diff
@@ -8,6 +8,13 @@ if BOARD_ACTINIUS_ICARUS_BEE || BOARD_ACTINIUS_ICARUS_BEE_NS
 config BOARD
 	default "actinius_icarus_bee"
 
+if SENSOR
+
+config I2C
+	default y
+
+endif # SENSOR
```

Comment:
ditto (same for all others)

Reply:
Fixed it, and also fixes for all similars.